File: MORM.Java/src/MORM/Java/Models/Venda/Transpagto.java
package MORM.Java.Models.Venda;

import java.util.Date;
import MORM.Java.Classes.*;
import MORM.Java.Classes.Mapping.*;

@TabelaAttribute(Nome = "TRANSPAGTO")
public class Transpagto extends CollectionItem
{
    @CampoAttribute(Campo = "ID_TRANSACAO", Tipo = CampoTipo.Key)
    public String Id_Transacao;
    @CampoAttribute(Campo = "NR_SEQ", Tipo = CampoTipo.Key)
    public Integer Nr_Seq;
    @CampoAttribute(Campo = "U_VERSION", Tipo = CampoTipo.Nul)
    public String U_Version;
    @CampoAttribute(Campo = "CD_OPERADOR", Tipo = CampoTipo.Req)
    public Integer Cd_Operador;
    @CampoAttribute(Campo = "DT_CADASTRO", Tipo = CampoTipo.Req)
    public Date Dt_Cadastro;
    @CampoAttribute(Campo = "ID_CAIXA", Tipo = CampoTipo.Req)
    public Integer Id_Caixa;
    @CampoAttribute(Campo = "TP_DOCUMENTO", Tipo = CampoTipo.Req)
    public Integer Tp_Documento;
    @CampoAttribute(Campo = "ID_HISTREL", Tipo = CampoTipo.Req)
    public Integer Id_Histrel;
    @CampoAttribute(Campo = "NR_PARCELA", Tipo = CampoTipo.Req)
    public Integer Nr_Parcela;
    @CampoAttribute(Campo = "NR_PARCELAS", Tipo = CampoTipo.Req)
    public Integer Nr_Parcelas;
    @CampoAttribute(Campo = "NR_DOCUMENTO", Tipo = CampoTipo.Nul)
    public Integer Nr_Documento;
    @CampoAttribute(Campo = "VL_DOCUMENTO", Tipo = CampoTipo.Req)
    public Float Vl_Documento;
    @CampoAttribute(Campo = "DT_VENCIMENTO", Tipo = CampoTipo.Req)
    public Date Dt_Vencimento;
    @CampoAttribute(Campo = "CD_AUTORIZACAO", Tipo = CampoTipo.Nul)
    public String Cd_Autorizacao;
    @CampoAttribute(Campo = "NR_NSU", Tipo = CampoTipo.Nul)
    public Integer Nr_Nsu;
    @CampoAttribute(Campo = "DS_REDETEF", Tipo = CampoTipo.Nul)
    public String Ds_Redetef;
    @CampoAttribute(Campo = "NM_OPERADORA", Tipo = CampoTipo.Nul)
    public String Nm_Operadora;
    @CampoAttribute(Campo = "NR_BANCO", Tipo = CampoTipo.Nul)
    public Integer Nr_Banco;
    @CampoAttribute(Campo = "NR_AGENCIA", Tipo = CampoTipo.Nul)
    public Integer Nr_Agencia;
    @CampoAttribute(Campo = "DS_CONTA", Tipo = CampoTipo.Nul)
    public String Ds_Conta;
    @CampoAttribute(Campo = "NR_CHEQUE", Tipo = CampoTipo.Nul)
    public Integer Nr_Cheque;
    @CampoAttribute(Campo = "DS_CMC7", Tipo = CampoTipo.Nul)
    public String Ds_Cmc7;
    @CampoAttribute(Campo = "TP_BAIXA", Tipo = CampoTipo.Nul)
    public Integer Tp_Baixa;
    @CampoAttribute(Campo = "CD_OPERBAIXA", Tipo = CampoTipo.Nul)
    public Integer Cd_Operbaixa;
    @CampoAttribute(Campo = "DT_BAIXA", Tipo = CampoTipo.Nul)
    public Date Dt_Baixa;
}

File: MORM.CSharp/Models/Venda/Transpagto.cs
using MORM.CSharp.Classes;
using System;

namespace MORM.CSharp.Models
{
    [Tabela("TRANSPAGTO")]
    public class Transpagto : CollectionItem
    {
        [Campo("ID_TRANSACAO", CampoTipo.Key)]
        public string Id_Transacao { get; set; }
        [Campo("NR_SEQ", CampoTipo.Key)]
        public int Nr_Seq { get; set; }
        [Campo("U_VERSION", CampoTipo.Nul)]
        public string U_Version { get; set; }
        [Campo("CD_OPERADOR", CampoTipo.Req)]
        public int Cd_Operador { get; set; }
        [Campo("DT_CADASTRO", CampoTipo.Req)]
        public DateTime Dt_Cadastro { get; set; }
        [Campo("ID_CAIXA", CampoTipo.Req)]
        public int Id_Caixa { get; set; }
        [Campo("TP_DOCUMENTO", CampoTipo.Req)]
        public int Tp_Documento { get; set; }
        [Campo("ID_HISTREL", CampoTipo.Req)]
        public int Id_Histrel { get; set; }
        [Campo("NR_PARCELA", CampoTipo.Req)]
        public int Nr_Parcela { get; set; }
        [Campo("NR_PARCELAS", CampoTipo.Req)]
        public int Nr_Parcelas { get; set; }
        [Campo("NR_DOCUMENTO", CampoTipo.Nul)]
        public int Nr_Documento { get; set; }
        [Campo("VL_DOCUMENTO", CampoTipo.Req)]
        public double Vl_Documento { get; set; }
        [Campo("DT_VENCIMENTO", CampoTipo.Req)]
        public DateTime Dt_Vencimento { get; set; }
        [Campo("CD_AUTORIZACAO", CampoTipo.Nul)]
        public string Cd_Autorizacao { get; set; }
        [Campo("NR_NSU", CampoTipo.Nul)]
        public int Nr_Nsu { get; set; }
        [Campo("DS_REDETEF", CampoTipo.Nul)]
        public string Ds_Redetef { get; set; }
        [Campo("NM_OPERADORA", CampoTipo.Nul)]
        public string Nm_Operadora { get; set; }
        [Campo("NR_BANCO", CampoTipo.Nul)]
        public int Nr_Banco { get; set; }
        [Campo("NR_AGENCIA", CampoTipo.Nul)]
        public int Nr_Agencia { get; set; }
        [Campo("DS_CONTA", CampoTipo.Nul)]
        public string Ds_Conta { get; set; }
        [Campo("NR_CHEQUE", CampoTipo.Nul)]
        public int Nr_Cheque { get; set; }
        [Campo("DS_CMC7", CampoTipo.Nul)]
        public string Ds_Cmc7 { get; set; }
        [Campo("TP_BAIXA", CampoTipo.Nul)]
        public int Tp_Baixa { get; set; }
        [Campo("CD_OPERBAIXA", CampoTipo.Nul)]
        public int Cd_Operbaixa { get; set; }
        [Campo("DT_BAIXA", CampoTipo.Nul)]
        public DateTime Dt_Baixa { get; set; }
    }
}